plugins/action/synchronize.py

```diff
@@ -204,7 +204,9 @@ def run(self, tmp=None, task_vars=None):
         # ssh paramiko docker buildah and local are fully supported transports.  Anything
         # else only works with delegate_to
         if delegate_to is None and self._connection.transport not in \
-                ('ssh', 'paramiko', 'local', 'docker', 'community.general.docker', 'community.docker.docker', 'buildah', 'containers.podman.buildah'):
+                ('ssh', 'paramiko', 'local', 'docker', 'community.general.docker',
+                 'community.docker.docker', 'buildah', 'containers.podman.buildah',
+                 'podman', 'containers.podman.podman'):
```

Comment:
I this we would want to avoid listing these in multiple places as we will face the same kind of bugs when we have to add a new transport or rename an existing one.

Reply:
Yes, Let us create a list of similar transports together - 

```python
@property
def docker_transport(self):
    return ['docker', 'community.general.docker', 'community.docker.docker']
...

if delegate_to is None and self._connection.transport not in self.docker_transport + [...]
...
```

Reply:
done with globals